scripts/test_env.py

```diff
@@ -0,0 +1,244 @@
+        style="{", 
+        format="{asctime} {levelname} \033[1m{funcName}:{lineno}\033[0m: {message}",)
+    # ISO 8601 timestamps without timezone
+    logging.Formatter.formatTime = (lambda self, record, datefmt=None:
+        datetime.datetime.fromtimestamp(record.created)
+        .isoformat(sep="T",timespec="microseconds"))
+    # Colored log levels
+    logging.addLevelName(logging.INFO, f"\033[32m{logging.getLevelName(logging.INFO)}\033[0m")
+    logging.addLevelName(logging.ERROR, f"\033[35m{logging.getLevelName(logging.ERROR)}\033[0m")
+    logging.addLevelName(logging.CRITICAL, f"\033[31m{logging.getLevelName(logging.CRITICAL)}\033[0m")
+    if args.command == "setup":
+        setup(args)
+    elif args.command == "cleanup":
+        cleanup(args)
+
+def setup(args):
+    logging.info(f"Starting iroha network with {args.n_peers} peers...")
+    try:
+        os.mkdir(args.out_dir)
+    except FileExistsError:
+        logging.error(f"Test directory `{args.out_dir}` already exists")
+        sys.exit(5)
+    try:
+        shutil.copy2(f"{args.root_dir}/target/debug/iroha_client_cli", args.out_dir)
+    except FileNotFoundError:
+        prompt_build_bin("iroha_client_cli", args.root_dir, args.out_dir)
+    with open(os.path.join(args.out_dir, "metadata.json"), "w") as f:
+        f.write('{"comment":{"String": "Hello Meta!"}}')
+    shutil.copy2(f"{args.root_dir}/configs/client/config.json", args.out_dir)
+    try:
+        shutil.copy2(f"{args.root_dir}/target/debug/kagami", args.out_dir)
+    except FileNotFoundError:
+        prompt_build_bin("kagami", args.root_dir, args.out_dir)
+
+    Network(args).run()
+
+def cleanup(args):
+    setup_dir = args.out_dir
+    logging.info("Killing peer processes...")
+    subprocess.run(["pkill", "-9", "iroha"])
+    logging.info(f"Cleaning up test directory `{setup_dir}`...")
+    shutil.rmtree(setup_dir)
+
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description=__doc__)
+
+    parser.add_argument("command", choices=["setup", "cleanup"],
+                        help="Available actions. The `setup` command will create and run \
+                        a new test environment with specified number of peers in a \
+                        directory of choice. This is meant to be run from `iroha` root dir. \
+                        The `cleanup` command will kill all peer processes \
+                        that were started by the `setup` command and remove the test directory")
+    parser.add_argument("n_peers", nargs="?", default=4, type=pos_int,
+                        help="Number of peers to bootstrap. \
+                        Defaults to 4. If setup was run with a custom number of peers, \
+                        the same number doesn't need to be provided to cleanup as \
+                        it kills all processes named `iroha`, so proper caution is advised")
+
+    parser.add_argument("--out_dir", "-o", default="./test", type=pathlib.Path,
```

Comment:
```suggestion
    parser.add_argument("--out-dir", "-o", default="./test", type=pathlib.Path,
```

`kebab-case` CLI args naming is more consistent with other CLIs in the repo.